pkg/docker/docker.go

```diff
@@ -347,16 +347,26 @@ func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 // and returns the image metadata
 func (d *stiDocker) CheckAndPullImage(name string) (*docker.Image, error) {
 	name = getImageName(name)
+	displayName := name
+
+	if glog.Is(3) == false {
+		// Summarize name for lower level log output
+		split := strings.Split(name, "@")
+		if len(split) > 1 && len(split[1]) > 18 {
```

Comment:
btw, there's a utility function here that parses the ref here:
https://github.com/openshift/origin/blob/master/pkg/image/api/helper.go#L133

We can't use it from here but maybe we can take the useful bits.